src/test/java/com/auth0/InvalidRequestExceptionMatcher.java

```diff
@@ -0,0 +1,47 @@
+package com.auth0;
+
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeMatcher;
+
+public class InvalidRequestExceptionMatcher extends TypeSafeMatcher<InvalidRequestException> {
```

Comment:
We don't seem to be using this class at all.
Is there a plan to use it in the future or is there something I may have missed?

Reply:
This is a test helper class. Not shipped in the release jar.